Rules/CommonScripts/HelpfulDeathTips.cfg

```diff
@@ -98,5 +97,20 @@ tips = To hide these tips, and other in-game help, hit F1.;
 		Knights can use their swords to destroy wooden blocks and doors, but not stone;
 		Doors and siege weapons are vulnerable to explosives.;
 		Bison can be tamed with burgers.;
-		Resources can be resupplied by changing classes in workshops, not just at the tent!;
+		You can get resupplies in class workshops, not just at the tent!;
 		You can highlight useful items for your class by holding the pickup key.;
+		As a knight, you can always jab the enemy after successfully slashing them to deal extra heart of damage.;
+		You should always have one transport tunnel near your tent and another one at your tower for quick transportation.;
```

Comment:
"at your tower" is a bit vague. I think it should instead suggest building a second tunnel "closer to the enemy".

Reply:
Also, it isn't essential to use tunnels for quick transport (ballistae exist), therefore, I recommend changing "You should always" to "It is advantageous to".